docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -115,6 +115,12 @@ catalog configuration.
     - Value of the new client (application) secret created
   * - ``hive.azure.adl-refresh-url``
     - OAuth 2.0 token endpoint url
+  * - ``hive.azure.adl-proxy-host``
+    - Proxy host and port in ``host:port`` format. Use this property to connect
+      to an ADLS endpoint via an internet proxy from a host from within an
+      HMS container. We **very strongly suggest** that this be used only when
```

Comment:
Remove "from a host from within an HMS container" since this is not specific to containers nor to the HMS from what I know.

Can you actually confirm where that proxy is used from these scenarios

* between coordinator and ADLS endpoint
* between worker and endpoint
* between HMS and endpoint
* anywhere else?

Reply:
I actually do think it is specific to containers. From slack convo: "we can get to the ADLS endpoint via an internet proxy from the host but not from within our HMS container. This is causing the HMS to fail to access ADLS."

@losipiuk - Should this mention instead a more generic connection than HMS containers specifically?

Reply:
That "HMS container" is a specific thing in the customer's environment, unrelated to this feature

Reply:
Remove the last sentence. Untrusted certificates is not related to this feature. The feature is very simple: it allows talking to ADLS using a SOCKS proxy. That's it. There are various reasons why someone might require the use of a proxy, but we don't need to mention them here. They wouldn't be using a proxy if they didn't have to.